props.json

```diff
@@ -31,6 +31,18 @@
         "title": "Night Forest With Aurora Sky by Architecture_Interior (CC BY)",
         "thumbnail": "https://igalia.github.io/wolvic-3D-environments/nightforest/nightforest.png",
         "payload": "https://igalia.github.io/wolvic-3D-environments/nightforest/nightforest.zip"
+      },
+      {
+        "value": "pretville_cinema",
+        "title": "Pretville Cinema by D. Savva and J. Guest (CC0)",
+        "thumbnail": "https://igalia.github.io/wolvic-test-environments/pretvillecinema/pretvillecinema.png",
+        "payload": "https://igalia.github.io/wolvic-test-environments/pretvillecinema/pretvillecinema.zip"
```

Comment:
By the way, should here be `https://igalia.github.io/wolvic-3D-environments` when https://github.com/Igalia/wolvic-3D-environments/pull/5 is merged?